src/Program.cs

```diff
@@ -51,7 +52,7 @@ await database.DefineContainer(Program.ContainerName, "/pk")
                             .Path("/*")
                             .Attach()
                     .Attach()
-                .CreateAsync(50000);
```

Comment:
@vjrantal 
I am inserting 1.5 million records but its showing this could you please help ??

![image](https://user-images.githubusercontent.com/37198555/90305809-f7874000-dee3-11ea-8d32-bd0d12292ca2.png)